tests/remote/variables.tf

```diff
@@ -1,31 +0,0 @@
-# Example usage in assert block
```

Comment:
These were moved to [tests/remote/test.tfvars](https://github.com/OpsLevel/terraform-provider-opslevel/pull/339/files#diff-e63027043413342b92a9ab5131b85463495382cff2976b3d990522550596da0f)